charts/tidb-cluster/values.yaml

```diff
@@ -554,6 +554,17 @@ scheduledBackup:
   # kubectl create secret generic s3-backup-secret --from-literal=access_key=<access-key> --from-literal=secret_key=<secret-key>
   # secretName: s3-backup-secret
 
+importer:
+  create: false
+  image: pingcap/tidb-lightning:v3.0.1
+  imagePullPolicy: IfNotPresent
+  storageClassName: local-storage
+  storage: 20Gi
```

Comment:
20 GB is likely not enough. It should be big enough to store the top 8 largest tables.